src/deploy/functions/services/auth.ts

```diff
@@ -83,9 +82,9 @@ export async function registerTrigger(
 
   if (!update) {
```

Comment:
I still don't think I understand why we only set these fields on function create instead of update.